### ISSUE TITLE
Support new "For In" loop syntax 

### ISSUE THREAD

**Issue by jinyulin** [robotframework/RIDE#1922]
Please support following new loop syntax, currently when perform save action in RIDE 1.7.4, the robot script will be automatic change to old syntax with '\'

FOR    ${animal}    IN    cat    dog    cow
    Keyword    ${animal}
    Another keyword
END

**Comment by Nyral** [comment]
How do you think this should be implemented?

**Comment by danielherrera** [comment]
I believe @jinyulin is referring to the FOR loop syntax that is available since Robot Framework 3.1 (already reported in #1826 )

Old syntax:
```
:FOR    ${animal}    IN    cat    dog    cow
\    Keyword    ${animal}
\    Another keyword
```

New syntax:
```
FOR    ${animal}    IN    cat    dog    cow
    Keyword    ${animal}
    Another keyword
END
```

https://github.com/robotframework/robotframework/blob/master/doc/releasenotes/rf-3.1.rst#for-loop-enhancements
